### DISCUSSION TITLE
Correct Helm chart repository URL for mirroring on Artifactory?

### DISCUSSION THREAD

**Discussion by edysli** [goharbor/harbor#19524] [Q&A]
I'd like to mirror on my Artifactory server a Helm chart repository hosted by Harbor. What's the correct URL path to give Artifactory?

**Comment by chlins** [comment]
Could you clarify your requirement, what you mean mirror the Artifactory, and BTW currently harbor does not support the `chartmuseum` since v2.8.

**Reply by edysli** [reply]
Sorry if that wasn't clear enough. What I meant is that I have an Artifactory instance, and in it I'd like to configure a remote Helm chart repository pointing to a chart repository hosted on a Harbor instance (which I do not control). The goal is to fetch Helm charts from Artifactory (with my Helm client) and Artifactory fetches charts from Harbor.

I have this currently configured on Artifactory, however the repository is empty, so I suspect the URL (pointing at Harbor) that I configured is wrong. Is there a URL path component to the chart repository on Harbor or should the URL simply point to the root of the web server, for example `https://harbor.example.com/`?
